ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/internalresponse/EthAccountsResultProvider.java

```diff
@@ -45,7 +48,28 @@ public EthAccountsResultProvider(final Supplier<Set<String>> addressesSupplier)
       throw new JsonRpcException(JsonRpcError.INVALID_PARAMS);
     }
 
-    return addressesSupplier.get().stream().sorted().collect(Collectors.toList());
+    final List<String> allAddress =
```

Comment:
Is there a test that verifies this behaviour, that the available addresses doesn't include addresses that have an invalid config file?

Reply:
Just updated Unit tests